include/envoy/http/codec.h

```diff
@@ -150,11 +150,17 @@ class ConnectionCallbacks {
 /**
  * A list of options that can be specified when creating a codec.
  */
-class CodecOptions {
+class CodecOptionFlags {
 public:
   static const uint64_t NoCompression = 0x1;
```

Comment:
It's not part of your change, but as long as you are in here, can you please rename ```NoCompression``` to ```DisableDynamicHPACKTable```. Don't bother changing the actual JSON config. We will fix that in v2 API.